requirements/default.txt

```diff
@@ -23,9 +23,9 @@ dennis==0.7 \
     --hash=sha256:fde267aeb03a8d49e3341e7f71a228c6748dce43b5c00f65581780d99b70b2d9
 
 # Web framework for Python projects of a certain age
```

Comment:
This comment is of a [certain age](https://github.com/mozilla/kuma/commit/e0d24246), but I hadn't noticed it before. Well done, @jwhitlock.